frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar_user.dart

```diff
@@ -92,10 +93,26 @@ class SidebarUser extends StatelessWidget {
 
   Widget _buildUserName(BuildContext context, MenuUserState state) {
     final String name = _userName(state.userProfile);
-    return FlowyText.medium(
-      name,
-      overflow: TextOverflow.ellipsis,
-      color: Theme.of(context).colorScheme.tertiary,
+    return GestureDetector(
+      onTap: () async {
+        final result = await OpenAIService.requestTextCompletion(
+          prompt: 'tell a story',
+          openAIKey: state.userProfile.openaiKey,
+          model: "gpt-3.5-turbo",
+        );
+        debugPrint("OPEN AI KEY:${state.userProfile.openaiKey}");
```

Comment:
Also remove this print. Actually, you can use tracing log.

Reply:
alright i'll just do it right away.

Reply:
yep